migri/backends/sqlite.py

```diff
@@ -8,7 +8,7 @@
 
 
 class SQLiteConnection(ConnectionBackend):
-    dialect = "sqlite"
+    _dialect = "sqlite"
```

Comment:
I think this may have been a mistake? The interface defined a property called `dialect` and which returned `_dialect`.